File: Review/Unit 3/MiniProject/PlaylistObjC/PlaylistObjC/Playlist.m
//
//  Playlist.m
//  PlaylistObjC
//
//  Created by Kaleo Kim on 4/21/16.
//  Copyright © 2016 Kaleo Kim. All rights reserved.
//

#import "Playlist.h"

@implementation Playlist

-(instancetype)initWithTitle:(NSString *)title songs:(NSMutableArray *)songs {
    
    self = [super init];
    if (self) {
        self.title = title;
        self.songs = songs;
    }
    return self;
}

@end

File: Week3/Mini Lesson/CoreDataPlaylist/Playlist.swift
//
//  Playlist.swift
//  CoreDataPlaylist
//
//  Created by Kaleo Kim on 2/15/16.
//  Copyright © 2016 Kaleo Kim. All rights reserved.
//

import Foundation
import CoreData


class Playlist: NSManagedObject {

    convenience init(name: String, context: NSManagedObjectContext = Stack.sharedStack.managedObjectContext) {
        let entity = NSEntityDescription.entityForName("Playlist", inManagedObjectContext: context)!
        
        self.init(entity: entity, insertIntoManagedObjectContext: context)
        
        self.name = name
    }

}
